fastlane/Helpers/Match.swift

```diff
@@ -26,17 +26,42 @@ enum Match {
             match(
                 type: type.value,
                 readonly: .userDefined(!isForce),
-                appIdentifier: appIdentifier,
+                appIdentifier: [environment.bundleId],
                 username: .userDefined(Constant.userName),
                 teamId: .userDefined(Constant.teamId),
                 gitUrl: Constant.matchURL,
                 force: .userDefined(isForce)
             )
         }
+        updateCodeSigning(type: type, environment: environment)
+    }
+    
+    static func updateCodeSigning(type: MatchType, environment: Environment) {
+        // Update Code signing from automatic to manual
+        updateCodeSigningSettings(
+            path: Constant.projectPath,
+            useAutomaticSigning: .userDefined(false),
+            targets: .userDefined([Constant.projectName]),
+            buildConfigurations: .userDefined(["\(type.buildConfiguration) \(environment.rawValue)"]),
+            codeSignIdentity: .userDefined(type.codeSignIdentity),
+            profileName: .userDefined("match \(type.method) \(environment.bundleId)")
+        )
     }
 }
 
 extension Match {
+    
+    enum Environment: String {
```

Comment:
I suggest returning array here in case some project has more than one bundle id for each environment.

Reply:
I still think its better to call `syncCodeSigning` for each environment seperately like we have been doing till now. Especially if we plan to support multiple apple accounts on the same project